modules/microshift-adding-other-packages-to-blueprint.adoc

```diff
@@ -20,7 +20,7 @@ Add the references for optional RPM packages to your OSTree blueprint to enable
 ----
 $ vi <microshift_blueprint.toml> <1>
```

Comment:
@ShaunaDiaz I should be adding underscore in the lines 21 and 30?
_<microshift_blueprint.toml>_
name = "_<microshift-additional-package-name>_"

Reply:
Looks like it, according to https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values.

Reply:
done